src/platform/plugins/shared/dashboard/public/dashboard_container/component/grid/use_layout_styles.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { transparentize, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+
+import { useMemo } from 'react';
+
+export const useLayoutStyles = () => {
+  const { euiTheme } = useEuiTheme();
+
+  const layoutStyles = useMemo(() => {
+    const getRadialGradient = (position: string) => {
+      return `radial-gradient(
+                circle at ${position},
+                ${euiTheme.colors.accentSecondary} 1px,
+                transparent 1px
+              )`;
+    };
+
+    return css`
+      .kbnGridRow--targeted {
+        background-position: top calc((var(--kbnGridGutterSize) / 2) * -1px) left
+          calc((var(--kbnGridGutterSize) / 2) * -1px);
+        background-size: calc((var(--kbnGridColumnWidth) + var(--kbnGridGutterSize)) * 1px)
+          calc((var(--kbnGridRowHeight) + var(--kbnGridGutterSize)) * 1px);
+        background-image: ${getRadialGradient('top left')}, ${getRadialGradient('top right')},
+          ${getRadialGradient('bottom left')}, ${getRadialGradient('bottom right')};
```

Comment:
The dot grid background is constructed like so: 

![image](https://github.com/user-attachments/assets/eb1d595c-6f93-4205-8a08-e728b14ecd6c)

You can see from the first image that each background image is sized to be the width + height of a single grid cell **plus** a single gutter size, and it is then shifted by **half** of the gutter size in order to be "centered" outside of the cell.